src/components/styledSelect/styledSelect.test.tsx

```diff
@@ -25,8 +26,6 @@ describe('StyledSelect', () => {
         />
       )
 
-      expect(wrapper).toBeTruthy()
```

Comment:
Not strictly related to this PR but it was part of the changes and I was going to change it in another PR, so it gets to stay.